### DISCUSSION TITLE
Spec generation no longer works in L11

### DISCUSSION THREAD

**Discussion by jonerickson** [tailflow/laravel-orion#303] [Bugs]
Due to L11 removing Doctrine DBAL, the package will no longer work generating schema. There are several deprecated methods now such as `getDoctrineSchemaManager`.

https://github.com/laravel/framework/pull/48864

Suggesting putting a constraint up to Illuminate 10.

**Comment by glennjacobs** [comment]
Seems to work for me in Laravel 11, presume this has been resolved.

**Reply by jonerickson** [reply]
What version of Laravel are you using?

**Reply by glennjacobs** [reply]
Laravel v11.26.0
Orion 2.22.5
